programs/buck_tool.py

```diff
@@ -1090,10 +1090,13 @@ def _get_java_args(self, version_uid, extra_default_options=None):
                 and os.environ.get("BUCK_DEBUG_MODE") != "0"
             ):
                 suspend = "n" if os.environ.get("BUCK_DEBUG_MODE") == "2" else "y"
+                port = "8888"
                 java_args.append(
                     "-agentlib:jdwp=transport=dt_socket,"
-                    "server=y,suspend=" + suspend + ",quiet=y,address=8888"
+                    "server=y,suspend=" + suspend + ",quiet=y,address=" + port
                 )
+                if (suspend=="y"):
+                    print("Waiting for debugger on port {}...".format(port))
```

Comment:
This should use logging.info/warning, not print to stdout.

Reply:
Done. 

There's another random use of print in that file, probably worth swapping out in a separate diff.